website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,174 +163,171 @@ The configuration options are listed below.
 
 The `IntermediatePKIPath` is the PKI engine used for storing the intermediate
-signed with the root certificate. The intermediate is used to sign all leaf
-certificates and Consul may periodically generate new intermediates for
+signed with the root certificate. The intermediate signs all leaf
+certificates, and Consul may periodically generate new intermediates for
 automatic rotation. Therefore, Consul requires write access to this path.
 
-If either path does not exist, then Consul will attempt to mount and
-initialize it. This requires additional privileges by the Vault token in use.
-If the paths already exist, Consul will use them as configured.
+If neither path exists, then Consul attempts to mount and
+initialize the PKI engine, which requires additional privileges of the Vault token in use.
+If the paths already exist, Consul uses them as configured.
 
-## Vault ACL Policies
+### Configure Vault ACL policies
 
-Vault PKI can be managed by either Consul or by Vault. If you want to manually create and tune the PKI secret engines used to store the root and intermediate certificates, use Vault Managed PKI Paths. If you want to have the PKI automatically managed for you, use Consul Managed PKI Paths.
+You can define an access control list (ACL) policy that assigns Vault PKI management to Consul or Vault. If you want to manually create and tune the PKI secret engines used to store the root and intermediate certificates, define a policy that uses Vault-managed PKI paths. If you want to automate PKI management, define a policy that uses Consul-managed PKI paths.
 
-### Vault Managed PKI Paths
+#### Define a policy for Vault-managed PKI paths
 
-The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
-Consul is granted read-only access to the PKI mount points and the Root CA, but is
-granted full control of the Intermediate or Leaf CA for Connect clients.
+You can define a Vault ACL policy that allows Consul to use pre-existing PKI paths in Vault. The policy grants Consul read-only access to the PKI mount points and the root CA and grants full control of the intermediate or leaf CA for Consul service mesh clients.
 
-This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
-Please update those placeholders to match the path values in the CA provider configuration.
+In the following examples, the path configurations use placeholder values for `RootPKIPath` and `IntermediatePKIPath`. Replace them to match the path values in the CA provider configuration.
 
-<CodeBlockConfig filename="vault-managed-pki-policy.hcl">
+1. Add the following rules to you policy file so that Consul can read the PKI mounts already created in Vault:
 
-```hcl
-# -----------------------------------------------------------------------------
-# Vault Managed PKI Mounts
-# -----------------------------------------------------------------------------
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
```

Comment:
Note to self: go back through all of these and make sure root vs intermediate is used in all the right places.

Reply:
After checking, I fixed one mistake where I mixed up root and intermediate.